pkg/rhoda/rhoda_test.go

```diff
@@ -18,165 +22,379 @@ import (
+					fmt.Println(inventory.Name)
+
+					if len(inventory.Status.Instances) > 0 {
+						testDBaaSConnection := dbaasv1alpha1.DBaaSConnection{
+							TypeMeta: meta.TypeMeta{
+								Kind:       "DBaaSConnection",
+								APIVersion: "dbaas.redhat.com/v1alpha1",
+							},
+							ObjectMeta: meta.ObjectMeta{
+								Name:      inventory.Status.Instances[0].Name,
+								Namespace: namespace,
+							},
+							Spec: dbaasv1alpha1.DBaaSConnectionSpec{
+								InventoryRef: dbaasv1alpha1.NamespacedName{
+									Namespace: namespace,
+									Name:      inventory.Name,
+								},
+								InstanceID: inventory.Status.Instances[0].InstanceID,
+							},
+						}
+
+						testDBaaSConnection.SetResourceVersion("")
+						Expect(client.Create(context.Background(), &testDBaaSConnection)).Should(Succeed())
+						By("checking DBaaSConnection status for: " + inventory.Status.Instances[0].Name)
+						Eventually(func() bool {
+							fmt.Println("checking DBaaSConnection status for: " + inventory.Status.Instances[0].Name)
+							err := client.Get(context.Background(), k8sClient.ObjectKey{
+								Namespace: namespace,
+								Name:      inventory.Status.Instances[0].Name,
+							}, &testDBaaSConnection)
+							Expect(err).NotTo(HaveOccurred())
+							if len(testDBaaSConnection.Status.Conditions) > 0 {
+								return testDBaaSConnection.Status.Conditions[0].Type == "ReadyForBinding" && testDBaaSConnection.Status.Conditions[0].Status == "True"
+							} else {
+								fmt.Println("testDBaaSConnection.Status.Conditions Len is 0")
+								return false
+							}
+						}, 60*time.Second, 5*time.Second).Should(BeTrue())
+					} else {
+						fmt.Println("No instances to connect")
+					}
+				} else {
+					fmt.Println("Inventory Status is not Ready for connection")
+					Expect(inventory.Status.Conditions[0].Status).To(Equal("True"))
 				}
-				//inventoryData, err := json.MarshalIndent(inventory, "", "  ")
-				//if err != nil {
-				//	fmt.Println("error:", err)
-				//}
-				//fmt.Print(string(inventoryData))
-				//	Eventually(isInventoryReady(c, inventory.ObjectMeta.Name, namespace), 30, 5).Should(BeTrue())
 			})
-		})
+		}
+	})
+
+	It("Console Login UI and pages access check", func() {
+		fmt.Println("Login")
+		ctx, cancel := chromedp.NewContext(context.Background())
+		defer cancel()
+		//Setting timeout
+		//ctx, cancel = context.WithTimeout(ctx, 5*time.Second)
+		//defer cancel()
+
+		var nodes []*cdp.Node
+		//selector for checking the Data Services button
+		selector := "#page-sidebar div ul li button"
+		url := "https://console-openshift-console.apps.rhoda-lab.51ty.p1.openshiftapps.com/dashboards"
+		// url:= "https://console-openshift-console.apps.rhoda-sp-prod.lue0.p1.openshiftapps.com/dashboards"
+		var data string
+
+		if err := chromedp.Run(ctx,
+			SetCookie("openshift-session-token", config.BearerToken, "console-openshift-console.apps.rhoda-lab.51ty.p1.openshiftapps.com", "/", false, false),
```

Comment:
If SetCookie is only used once to make the code more readable, it should be `SetOpenShiftCookie` and you would only pass it the two values that are not always the same